tests/ociswrapper/wrapper/handlers/handler.go

```diff
@@ -113,7 +113,8 @@ func RollbackHandler(res http.ResponseWriter, req *http.Request) {
 
 	var message string
 	ocis.EnvConfigs = []string{}
-	success, _ := ocis.Restart(os.Environ())
+
+	success, _ := ocis.Restart([]string{})
```

Comment:
is this change required?

Reply:
Yes, of course. Otherwise, duplication of the OS environment will occur. The current OCIS wrapper version has an issue with duplicate environment variables.

Reply:
good stuff :+1: